sdk/storage/storage-blob/src/BlobClient.ts

```diff
@@ -1300,6 +1310,33 @@ export class BlobClient extends StorageClient {
     }
   }
 
+  /**
+   *
+   * @param copySource
+   * @param options
+   */
+  public async beginCopyFromURL(copySource: string, options: BlobBeginCopyFromURLOptions = {}) {
```

Comment:
>beginCopyFromURL [](start = 15, length = 16)

Can customer get raw response from  beginCopyFromURL ? Some customers like Yammer likes to get request_id for tracking & logging.

Reply:
Right now if the operation fails, they'll get the RestError that includes the raw response. If it succeeds, we don't have a way to expose it directly (aside from enabling logging). We could make changes to core-lro to support this in the future, we'll probably want to expose raw responses in a standardized way when using pollers.

Reply:
As prior art, .net's Operation<T> has `getRawResponse()` which returns the last response the server gave to the poller. I'd probably call the property differently, but it should be fairly trivial to expose this on LRO later.

Reply:
Sounds good. Most of storage APIs provide ability to get raw response. This helps for advanced users. Can we create an issue for task tracking?

Reply:
@ramya-rao-a  Created an item for tracking https://github.com/Azure/azure-sdk-for-js/issues/5801